Library/Homebrew/cmd/test-bot.rb

```diff
@@ -520,7 +521,7 @@ def formula formula_name
               test "brew", "uninstall", "--force", *unchanged_build_dependencies
               unchanged_dependencies -= unchanged_build_dependencies
             end
-            test "brew", "install", bottle_filename
+            test "brew", "install", "--force-bottle", bottle_filename
```

Comment:
Welp. You are right. The only way it verifies the local checksum with `--force-bottle` is if you do something like this:

```ruby
  bottle do
    root_url "file:///Users/Dominyk/Downloads"
    revision 2
    sha256 "2c04bd2197ededcfe295a0366c0ee4c1322708e50f9cebef084460db960f78cc" => :yosemite
  end
```

```
~> brew install -v --force-bottle a2ps
==> Downloading file:///Users/Dominyk/Downloads/a2ps-4.14.yosemite.bottle.2.tar.gz
Already downloaded: /usr/local/var/homebrew/cache/a2ps-4.14.yosemite.bottle.2.tar.gz
==> Verifying a2ps-4.14.yosemite.bottle.2.tar.gz checksum
==> Pouring a2ps-4.14.yosemite.bottle.2.tar.gz
tar xf /usr/local/var/homebrew/cache/a2ps-4.14.yosemite.bottle.2.tar.gz
==> Finishing up
```

Otherwise:

```
~> brew install -v --force-bottle ./a2ps-4.14.yosemite.bottle.2.tar.gz
==> Pouring a2ps-4.14.yosemite.bottle.2.tar.gz
tar xf /Users/Dominyk/Downloads/a2ps-4.14.yosemite.bottle.2.tar.gz
```